recipes/fmilib/all/patches/2.4.1-009-make-project-pure-c.patch

```diff
@@ -0,0 +1,39 @@
+From dcf9863528e64edcad83ff0f9242b94b14df1705 Mon Sep 17 00:00:00 2001
```

Comment:
This patch makes sure that the library can be built with a C compiler only, which the upstream cannot, since it builds third party libraries that require CMake to look for C++ compiler. By applying this patch, we can remove libcxx and cppstd settings from the recipe. Looking at the upstream code, fmilib 3 will require c++ (at least building tests), so it can be dropped when fmilib 3 is released, and therefore no need to maintain it beyond fmilib 2.4.1.